src/EnqueueSerializer.php

```diff
@@ -0,0 +1,60 @@
+<?php
+namespace Prooph\ServiceBus\Message\Enqueue;
+
+use Enqueue\Util\JSON;
+use Prooph\Common\Messaging\Message;
+use Prooph\Common\Messaging\MessageConverter;
+use Prooph\Common\Messaging\MessageDataAssertion;
+use Prooph\Common\Messaging\MessageFactory;
+
+class EnqueueSerializer
```

Comment:
move this whole code to the message producer, or is it used somewhere else as well?